app/services/person_constructor.rb

```diff
@@ -0,0 +1,82 @@
+class PersonConstructor
+  KEY_NAME_MAPPINGS = {
+    address: :address_1,
+    zip: :zip_code,
+  }
+  OLD_REMOTE_PARAMS = [
+    remote_fields: PersonWithRemoteFields::REMOTE_PARAMS
+  ]
+
+  def self.permitted_params
```

Comment:
I agree with the idea of using methods or attributes over variables, but constants are not the same.  Here we have an immutable set that will never be changed by our app - essentially the definition of a constant.